### PR TITLE
Adds reset() method

### PR DESCRIPTION
**Pull request by emersion**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -28,4 +28,10 @@ Controller.prototype.update = function(current_value) {
   return (this.k_p*error) + (this.k_i * this.sumError) + (this.k_d * dError);
 };
 
-module.exports = Controller;
\ No newline at end of file
+Controller.prototype.reset = functrion() {
```
Comment (Philmod) [inline lib/index.js]: typo in `function`

**Comment by Philmod** [comment]
Please write a test.

File: lib/index.js (hunk 1 above)
Comment (emersion) [inline lib/index.js]: Ow, sorry :-/

**Comment by emersion** [comment]
Done!

**Pull request merged**